src/tv2-common/migrations/sourceLayers.ts

```diff
@@ -0,0 +1,39 @@
+import { ISourceLayer, MigrationContextShowStyle, MigrationStepShowStyle } from '@tv2media/blueprints-integration'
+import { WithValuesOfTypes } from 'tv2-common'
+import _ = require('underscore')
+
+export type WithPrimitiveValues<T> = WithValuesOfTypes<T, string | boolean | number | null>
+
+export function SetSourceLayerProperties(
+	versionStr: string,
+	sourceLayerId: string,
+	newValues: Omit<Partial<WithPrimitiveValues<ISourceLayer>>, '_id'>
+): MigrationStepShowStyle {
+	return {
+		id: `${versionStr}.setSourceLayerProperties.${Object.keys(newValues).join('_')}.${sourceLayerId}`,
+		version: versionStr,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const sourceLayer = context.getSourceLayer(sourceLayerId)
+
+			if (!sourceLayer) {
+				// nothing to migrate
+				// getSourceLayerDefaultsMigrationSteps should create this layer later
+				return false
+			}
+
+			return !_.isMatch(sourceLayer, newValues)
+		},
+		migrate: (context: MigrationContextShowStyle) => {
+			context.updateSourceLayer(sourceLayerId, newValues)
+		}
+	}
+}
+
+export function SetSourceLayerName(
+	versionStr: string,
+	sourceLayerId: string,
+	newValue: string
```

Comment:
Minor thing. Since the method is called "SetSourceLayerName" then the parameter "newValue" would always be the new name, so it might as well be called "newName" instead.